ballerina/tests/tests.bal

```diff
@@ -0,0 +1,285 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/oauth2;
+import ballerina/test;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+configurable string serviceUrl = ?;
```

Comment:
No need to define the `serviceUrl` here as the client already have a default value to the `serviceUrl`.
```suggestion
```